lib/dribbble/project.rb

```diff
@@ -1,11 +1,20 @@
+# frozen_string_literal: true
+
 require 'dribbble/utils/findable'
+# require 'dribbble/utils/creatable'
+# require 'dribbble/utils/updatable'
+# require 'dribbble/utils/deletable'
 
 module Dribbble
   class Project < Dribbble::Base
+    # TODO: enable these once the GET /projects/:id endpoint gets available!! https://developer.dribbble.com/v2/projects/
```

Comment:
Reading the documentation today, they seem to be available right?

Reply:
Hmm. Indeed it seems the endpoints have been added. I'll make the change.